frontend/packages/integration-tests-cypress/tests/app/multi-cluster.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { checkErrors } from '../../support';
+import { projectDropdown } from '../../views/common';
+import { guidedTour } from '../../views/guided-tour';
+import { listPage } from '../../views/list-page';
+import { nav } from '../../views/nav';
+
+const MULTI_CLUSTER_ARRAY = ['apollo-cluster', 'bravo-cluster'];
+describe('Multi Cluster Perspective Test', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.switcher.changePerspectiveTo('Developer');
+    nav.sidenav.switcher.shouldHaveText('Developer');
+    guidedTour.close();
+    cy.window().then((win: any) => {
+      win.SERVER_FLAGS.clusters = MULTI_CLUSTER_ARRAY;
+      nav.sidenav.switcher.changePerspectiveTo('Administrator');
```

Comment:
Worth a comment here explaining that we switch perspectives to force a re-render.